### PR TITLE
Add precision about how stats are calculated

### PR DESCRIPTION
**Pull request by henri-tremblay**

### CODE REVIEW

**Comment by henri-tremblay** [comment]
You can revise but I would prefer waiting for feedback from the doc team before merging.

File: docs/src/docs/asciidoc/user/management.adoc [hunk 1]
```diff
@@ -102,32 +102,40 @@ This way, all managed objects get registered into a common `ManagementRegistry`
 
 == Rules for statistics calculation
 
-This table describes the impact of each cache method on the statistics. It is compatible with
+This table describes if yes or no, a cache method can impact a given statistic. It is compatible with
```
Comment (ChrisBradley001) [inline docs/src/docs/asciidoc/user/management.adoc]: This describes whether or not a cache method can impact a given statistic. It is compatible with

File: docs/src/docs/asciidoc/user/management.adoc [hunk 2]
```diff
@@ -102,32 +102,40 @@ This way, all managed objects get registered into a common `ManagementRegistry`
...
+|get            |Yes|Yes |No |No    |No     |Hit when the entry is found, miss otherwise
+|getAll         |Yes|Yes |No |No    |No     |Like `get` but calculated per entry
+|getAndPut      |Yes|Yes |Yes|Yes   |No     |JSR107 specific. Hit and update when the entry is found, miss otherwise. Always put
+|getAndRemove   |Yes|Yes |No |No    |Yes    |JSR107 specific. Hit and remove when the entry is found, miss otherwise
+|getAndReplace  |Yes|Yes |Yes|Yes   |No     |JSR107 specific. Hit, put and update when the entry is found, miss otherwise
+|invoke         |Yes|Yes |Yes|Yes   |Yes    |JSR107 specific. Depends on how the `EntryProcessor` modifies the `MutableEntry`
+|invokeAll      |Yes|Yes |Yes|Yes   |Yes    |JSR107 specific. Like `invoke` but calculated per entry
+|iterator       |Yes|No  |No |No    |Yes    |Will hit each iterated entry and count a removal on `Iterator.remove`
+|loadAll        |No |No  |No |No    |No     |JSR107 specific. Background loading that has no impact on stats
+|put            |No |No  |Yes|Yes   |No     |Put an entry whether is was existing or not
```
Comment (ChrisBradley001) [inline docs/src/docs/asciidoc/user/management.adoc]: Put an entry whether it already existed or not

File: docs/src/docs/asciidoc/user/management.adoc [hunk 3]
```diff
@@ -102,32 +102,40 @@ This way, all managed objects get registered into a common `ManagementRegistry`
...
+|putAll         |No |No  |Yes|Yes   |No     |Like `put` but calculated per entry
+|putIfAbsent    |Yes|Yes |Yes|No    |No     |Will hit is the entry exists. Will put and miss if it doesn't
```
Comment (ChrisBradley001) [inline docs/src/docs/asciidoc/user/management.adoc]: Will hit if the entry exists. Will put and miss if it doesn't

Comment (henri-tremblay) [inline docs/src/docs/asciidoc/user/management.adoc]: Done

File: docs/src/docs/asciidoc/user/management.adoc (hunk 2 above)
Comment (henri-tremblay) [inline docs/src/docs/asciidoc/user/management.adoc]: done

File: docs/src/docs/asciidoc/user/management.adoc (hunk 1 above)
Comment (henri-tremblay) [inline docs/src/docs/asciidoc/user/management.adoc]: done

**Pull request merged**